src/main/java/com/microfocus/application/automation/tools/octane/events/AbstractBuildListenerOctaneImpl.java

```diff
@@ -126,7 +128,8 @@ private void publishFinishEvent(AbstractBuild build) {
 					.setParameters(ParameterProcessors.getInstances(build))
 					.setResult(BuildHandlerUtils.translateRunResult(build))
 					.setDuration(build.getDuration())
-					.setTestResultExpected(hasTests);
+					.setTestResultExpected(hasTests)
+					.setEnvironmentOutputtedParameters(getOutputEnvironmentParams(build));
```

Comment:
Can you please use the helper inside the set method, it more readable like other setters instead of adding it on import

Reply:
done